docker/webserver/generate_quic_host_key.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+# See https://nginx.org/en/docs/http/ngx_http_v3_module.html#quic_host_key
+#    Sets a file with the secret key used to encrypt stateless reset and address validation tokens.
+#    By default, a random key is generated on each reload. Tokens generated with old keys are not accepted.
+
+# The default NGX_QUIC_DEFAULT_HOST_KEY_LEN is 32 bytes (ngx_event_quic.h)
+# Since reloads happen due to Certbot, set a static persistent host key per release.
```

Comment:
This regenerates the key on restart though, right? Certbot reloads every 60 days? We might release more often than that. If it was a concern to regenerate the key every 60 days, we need this to be persistent rather than on start.

We do also have reloads when adding batch users, so reloads are more frequent - that's just not mentioned as a reason here.

Reply:
You are correct, there are more reasons, I will add them.
My idea was a new release or container restart might be a good time to regenerate the key, this will invalidate old [QUIC Address Validation tokens](https://datatracker.ietf.org/doc/html/rfc9000#name-address-validation), the alternative would be persistent storage, but I'm unsure if it is needed to periodically cycle the key.
So all in all, is just updating the comment enough?

Reply:
Yes, just for future clarity :)